libs/agent-scripts/cloudbroker/grid/check_node_version.py

```diff
@@ -0,0 +1,63 @@
+from JumpScale import j
+import yaml
+
+descr = """
+This script checks node version, if it out of date, trigger update_node jumpscript to update it.
+"""
+organization = "greenitglobe"
+category = "cloudbroker"
+version = "1.0"
+period = "0 0 12 1/2 * ? *"  # every 2 days at 12:00
+startatboot = True
+enable = True
+async = True
+roles = ["node"]
+queue = "process"
+log = True
+
+scl = j.clients.osis.getNamespace("system")
```

Comment:
there should be no code in the module itself (this gets execute by the agentcontroller)